config/environment.js

```diff
@@ -144,6 +144,11 @@ module.exports = function(environment) {
             popularNode,
             noteworthyNode,
         },
+        featureFlags: { // default flags (whether they be switches, flags, or polls) go here with default value.
+            support: 'ember_support_page',
+            dashboard: 'ember_dashboard_page',
```

Comment:
I think these flag names are a little too generic. Perhaps `route_support` and `route_dashboard`?

Reply:
Would it be too horrible to just use the names from the API and no mapping?